src/app-gocardless/banks/virgin_nrnbgb22.js

```diff
@@ -0,0 +1,49 @@
+import Fallback from './integration-bank.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  institutionIds: ['VIRGIN_NRNBGB22'],
+
+  accessValidForDays: 90,
+
+  normalizeAccount(account) {
+    return Fallback.normalizeAccount(account);
+  },
+
+  normalizeTransaction(transaction, booked) {
+    const transferPrefixes = ['MOB', 'FPS'];
+    const methodRegex = /(Card|WLT)\s\d*/;
```

Comment:
Should this be anchored to the start of the string?

Should it be `\d+` instead of 0 or more digits?

Reply:
You're right, fixed